server/src/socobouser/handlers/SocoboUserRoles.handler.ts

```diff
@@ -0,0 +1,40 @@
+import { Request, Response } from "express";
+import { IDatabase } from "pg-promise/typescript/pg-promise";
+import { DbExtensions } from "../../app/index";
+import { SocoboUserRole } from "../models/SocoboUserRole";
+
+export class SocoboUserRolesHandler {
+  private _db: IDatabase<DbExtensions>&DbExtensions;
+
+  constructor (db: any) {
+    this._db = db;
+  }
+
+  public getAll = (req: Request, res: Response): void => {
+    this._db.socobouserRoles.getAll()
+      .then((result: SocoboUserRole[]) => res.status(200).json(result))
+      .catch((e: any) => res.status(e.statusCode).json(e.forResponse()));
+  }
+
+  public getById = (req: Request, res: Response): void => {
+    const roleId: number = +req.params.id;
+    this._db.socobouserRoles.getById(roleId)
+      .then((result: SocoboUserRole) => res.status(200).json(result))
+      .catch((e: any) => res.status(e.statusCode).json(e.forResponse()));
+  }
+
+  public updateById = (req: Request, res: Response): void => {
+    const roleId: number = +req.params.id;
+    const fieldValue: string = req.body.fieldValue;
+    this._db.socobouserRoles.updateById(roleId, fieldValue)
+      .then((result: SocoboUserRole) => res.status(200).json(result))
+      .catch((e: any) => res.status(e.statusCode).json(e.forResponse()));
+  }
+
+  public deleteById = (req: Request, res: Response): void => {
```

Comment:
Where do we use the roles values from the database? I'm not sure if I understand the purpose of storing roles in the db. Couldn't we simply store the name of the role as String into the user relation?